music/src/core/visualizer.ts

```diff
@@ -330,7 +330,13 @@ export class PianoRollSVGVisualizer extends BaseVisualizer {
       sequence: INoteSequence, svg: SVGSVGElement,
       config: VisualizerConfig = {}) {
     super(sequence, config);
-
+        
+    //Return error if element isn't svg
+    if (!(svg instanceof SVGSVGElement)) {
```

Comment:
What do you think about being a bit more aggressive and throwing an Error instead? So something like `throw new Error('mm.PianoRollSVGVisualizer requires an <svg> element to display the visualization');`

Reply:
Sure!